test/get_previous_releases.py

```diff
@@ -107,40 +107,40 @@ def download_binary(tag, args) -> int:
     platform = args.platform
     if tag < "v23" and platform in ["x86_64-apple-darwin", "arm64-apple-darwin"]:
         platform = "osx64"
-    tarball = 'bitcoin-{tag}-{platform}.tar.gz'.format(
+    archive = 'bitcoin-{tag}-{platform}.tar.gz'.format(
         tag=tag[1:], platform=platform)
-    tarballUrl = 'https://bitcoincore.org/{bin_path}/{tarball}'.format(
-        bin_path=bin_path, tarball=tarball)
+    archiveUrl = 'https://bitcoincore.org/{bin_path}/{archive}'.format(
+        bin_path=bin_path, archive=archive)
 
-    print('Fetching: {tarballUrl}'.format(tarballUrl=tarballUrl))
+    print('Fetching: {archiveUrl}'.format(archiveUrl=archiveUrl))
 
-    ret = subprocess.run(['curl', '--fail', '--remote-name', tarballUrl]).returncode
+    ret = subprocess.run(['curl', '--fail', '--remote-name', archiveUrl]).returncode
     if ret:
         return ret
 
     hasher = hashlib.sha256()
-    with open(tarball, "rb") as afile:
+    with open(archive, "rb") as afile:
         hasher.update(afile.read())
-    tarballHash = hasher.hexdigest()
+    archiveHash = hasher.hexdigest()
 
-    if tarballHash not in SHA256_SUMS or SHA256_SUMS[tarballHash]['tarball'] != tarball:
-        if tarball in [v['tarball'] for v in SHA256_SUMS.values()]:
+    if archiveHash not in SHA256_SUMS or SHA256_SUMS[archiveHash]['archive'] != archive:
+        if archive in [v['archive'] for v in SHA256_SUMS.values()]:
             print("Checksum did not match")
             return 1
 
         print("Checksum for given version doesn't exist")
         return 1
     print("Checksum matched")
 
-    # Extract tarball
-    ret = subprocess.run(['tar', '-zxf', tarball, '-C', tag,
+    # Extract archive
+    ret = subprocess.run(['tar', '-zxf', archive, '-C', tag,
                           '--strip-components=1',
                           'bitcoin-{tag}'.format(tag=tag[1:])]).returncode
     if ret != 0:
-        print(f"Failed to extract the {tag} tarball")
+        print(f"Failed to extract the {tag} archive")
         return ret
 
-    Path(tarball).unlink()
+    Path(archive).unlink()
 
     if tag >= "v23" and platform == "arm64-apple-darwin":
```

Comment:
1a1b478ca31be1f670754a47da17863271e46b7b: missed one occurance of `platform`, see #32922